### PR TITLE
Updated from the review

### LINKED ISSUE #229: [Docu] View Exam Formatting
image

The note seems a bit weird to have `blocks` everywhere

### LINKED ISSUE #238: [Exam] examslist should be called listexams
To keep with consistency with the other list commands

### LINKED ISSUE #248: [Docu] UG examslist should come before delete exams
Because it is used in the examples of delete exams

### PR DESCRIPTION
**Pull request by micklwl**
Resolves #229 
Resolves #238 
Resolves #248 
Exam is now back to being unique in ALL fields and reverted before change was made due to #200.
Formatted presentation of an exam.
Removed some magic numbers.

### CODE REVIEW

File: src/seedu/addressbook/ui/MainWindow.java [hunk 1]
```diff
@@ -186,14 +192,14 @@ void onCommand(ActionEvent event) {
...
-            // If this error occurs, data between addressbook and storage is likely to be desynced,
+            // If this error occurs, data between the app and storage is likely to be desynced,
             // so we force close the app to ensure synchronisation.
             final int delayTime = 3000;
             exitApp(delayTime);
         }
     }
 
-    /** Exits the app after a given delay*/
+    /** Exits the app after a set delay of 500ms*/
```
Comment (0WN463) [inline src/seedu/addressbook/ui/MainWindow.java]: Seems like a pretty lame comment though, considering that it needs to be changed if we change the delay, which makes it a "bad comment"

Comment (micklwl) [reply src/seedu/addressbook/ui/MainWindow.java]: its a magic number :)

Comment (0WN463) [reply src/seedu/addressbook/ui/MainWindow.java]: Its not, its under a named variable

Comment (micklwl) [reply src/seedu/addressbook/ui/MainWindow.java]: I'll remove the 500 then. I need to update my code too.

**Pull request merged**